engine/dlib/src/dlib/socket_posix.cpp

```diff
@@ -202,7 +207,7 @@ namespace dmSocket
         *socket = sock;
         if (sock >= 0)
         {
-#if defined(__MACH__)
+#if defined(__MACH__) && !defined(__EMSCRIPTEN__)
```

Comment:
Isn't it already not `__EMSCRIPTEN__` because it is `__MACH__` ?

Reply:
You are right! Good catch!